cannon/mipsevm/memory/memory64_test.go

```diff
@@ -144,41 +144,27 @@ func TestMemory64ReadWrite(t *testing.T) {
 		m := NewMemory()
 		m.SetWord(16, 0xAABBCCDD_EEFF1122)
 		require.Equal(t, Word(0xAABBCCDD_EEFF1122), m.GetWord(16))
-		require.Equal(t, uint32(0xAABBCCDD), m.GetUint32(16))
-		require.Equal(t, uint32(0xEEFF1122), m.GetUint32(20))
+		require.Equal(t, Word(0xAABBCCDD_EEFF1122), m.GetWord(16))
 		m.SetWord(16, 0xAABB1CDD_EEFF1122)
 		require.Equal(t, Word(0xAABB1CDD_EEFF1122), m.GetWord(16))
-		require.Equal(t, uint32(0xAABB1CDD), m.GetUint32(16))
-		require.Equal(t, uint32(0xEEFF1122), m.GetUint32(20))
+		require.Equal(t, Word(0xAABB1CDD_EEFF1122), m.GetWord(16))
 		m.SetWord(16, 0xAABB1CDD_EEFF1123)
 		require.Equal(t, Word(0xAABB1CDD_EEFF1123), m.GetWord(16))
-		require.Equal(t, uint32(0xAABB1CDD), m.GetUint32(16))
-		require.Equal(t, uint32(0xEEFF1123), m.GetUint32(20))
+		require.Equal(t, Word(0xAABB1CDD_EEFF1123), m.GetWord(16))
```

Comment:
Looks like we can just cut these GetUint32 calls - we were already calling GetWord so we have duplicate checks now.